Index: /usr/local/emhttp/auth-request.php
===================================================================
--- /usr/local/emhttp/auth-request.php	original
+++ /usr/local/emhttp/auth-request.php	modified
@@ -13,10 +13,11 @@
   }
   session_write_close();
 }
 
 $arrWhitelist = [
+  '/webGui/images/UN-logotype-gradient.svg',
   '/webGui/styles/clear-sans-bold-italic.eot',
   '/webGui/styles/clear-sans-bold-italic.woff',
   '/webGui/styles/clear-sans-bold-italic.ttf',
   '/webGui/styles/clear-sans-bold-italic.svg',
   '/webGui/styles/clear-sans-bold.eot',
