Index: /usr/local/emhttp/plugins/dynamix/include/DefaultPageLayout.php
===================================================================
--- /usr/local/emhttp/plugins/dynamix/include/DefaultPageLayout.php	original
+++ /usr/local/emhttp/plugins/dynamix/include/DefaultPageLayout.php	modified
@@ -602,20 +602,11 @@
 }
 
 function openNotifier() {
   $.post('/webGui/include/Notify.php',{cmd:'get',csrf_token:csrf_token},function(msg) {
     $.each($.parseJSON(msg), function(i, notify){
-      $.jGrowl(notify.subject+'<br>'+notify.description,{
-        group: notify.importance,
-        header: notify.event+': '+notify.timestamp,
-        theme: notify.file,
-        sticky: true,
-        beforeOpen: function(e,m,o){if ($('div.jGrowl-notification').hasClass(notify.file)) return(false);},
-        afterOpen: function(e,m,o){if (notify.link) $(e).css('cursor','pointer');},
-        click: function(e,m,o){if (notify.link) location.replace(notify.link);},
-        close: function(e,m,o){$.post('/webGui/include/Notify.php',{cmd:'archive',file:notify.file,csrf_token:csrf_token});}
-      });
+      
     });
   });
 }
 
 function closeNotifier() {
@@ -748,12 +739,12 @@
   }
   // create list of nchan scripts to be started
   if (isset($button['Nchan'])) nchan_merge($button['root'], $button['Nchan']);
 }
 
-echo "<div class='nav-user show'><a id='board' href='#' class='hand'><b id='bell' class='icon-u-bell system'></b></a></div>";
 
+
 if ($themes2) echo "</div>";
 echo "</div></div>";
 foreach ($buttons as $button) {
   annotate($button['file']);
   // include page specific stylesheets (if existing)
@@ -960,26 +951,18 @@
         case 'warning': bell2++; break;
         case 'normal' : bell3++; break;
       }
 <?if ($notify['display']==0):?>
       if (notify.show) {
-        $.jGrowl(notify.subject+'<br>'+notify.description,{
-          group: notify.importance,
-          header: notify.event+': '+notify.timestamp,
-          theme: notify.file,
-          beforeOpen: function(e,m,o){if ($('div.jGrowl-notification').hasClass(notify.file)) return(false);},
-          afterOpen: function(e,m,o){if (notify.link) $(e).css('cursor','pointer');},
-          click: function(e,m,o){if (notify.link) location.replace(notify.link);},
-          close: function(e,m,o){$.post('/webGui/include/Notify.php',{cmd:'hide',file:"<?=$notify['path'].'/unread/'?>"+notify.file,csrf_token:csrf_token}<?if ($notify['life']==0):?>,function(){$.post('/webGui/include/Notify.php',{cmd:'archive',file:notify.file,csrf_token:csrf_token});}<?endif;?>);}
-        });
+        
       }
 <?endif;?>
     });
-    $('#bell').removeClass('red-orb yellow-orb green-orb').prop('title',"<?=_('Alerts')?> ["+bell1+']\n'+"<?=_('Warnings')?> ["+bell2+']\n'+"<?=_('Notices')?> ["+bell3+']');
-    if (bell1) $('#bell').addClass('red-orb'); else
-    if (bell2) $('#bell').addClass('yellow-orb'); else
-    if (bell3) $('#bell').addClass('green-orb');
+
+
+
+
     break;
   }
 });
 
 <?if ($wlan0):?>
@@ -1363,7 +1346,8 @@
     }
   }
 }
 <?endif;?>
 </script>
+<uui-toaster rich-colors close-button position="<?= ($notify['position'] === 'center') ? 'top-center' : $notify['position'] ?>"></uui-toaster>
 </body>
 </html>
