Index: /usr/local/emhttp/plugins/dynamix/DisplaySettings.page
===================================================================
--- /usr/local/emhttp/plugins/dynamix/DisplaySettings.page	original
+++ /usr/local/emhttp/plugins/dynamix/DisplaySettings.page	modified
@@ -134,11 +134,11 @@
   </select>
 
 :display_width_help:
 
 _(Language)_:
-: <select name="locale" class="fixed" onchange="updateDirection(this.value)">
+: <select name="locale" onchange="updateDirection(this.value)">
 <?echo mk_option($display['locale'], "","English");
   foreach (glob("$plugins/lang-*.xml",GLOB_NOSORT) as $xml_file) {
     $lang = language('Language', $xml_file);
     $home = language('LanguageLocal', $xml_file);
     $name = language('LanguagePack', $xml_file);
