Index: /usr/local/emhttp/plugins/dynamix/Notifications.page
===================================================================
--- /usr/local/emhttp/plugins/dynamix/Notifications.page	original
+++ /usr/local/emhttp/plugins/dynamix/Notifications.page	modified
@@ -127,33 +127,36 @@
   <?=mk_option($notify['position'], "bottom-right", _("bottom-right"))?>
   <?=mk_option($notify['position'], "center", _("center"))?>
   </select>
 
 :notifications_display_position_help:
+:
+ _(Stack notifications)_:
+: <select name="expand">
+  <?=mk_option($notify['expand'] ?? 'true', "true", _("Yes"))?>
+  <?=mk_option($notify['expand'] ?? 'true', "false", _("No"))?>
+  </select>
 
-_(Auto-close)_ (_(seconds)_):
-: <input type="number" name="life" class="a" min="0" max="60" value="<?=$notify['life']?>"> _(a value of zero means no automatic closure)_
+:notifications_stack_help:
 
-:notifications_auto_close_help:
+_(Duration)_:
+: <input type="number" name="duration" value="<?=$notify['duration'] ?? 5000?>" min="1000" step="500">
 
-_(Date format)_:
-: <select name="date" class="a">
-  <?=mk_option($notify['date'], "d-m-Y", _("DD-MM-YYYY"))?>
-  <?=mk_option($notify['date'], "m-d-Y", _("MM-DD-YYYY"))?>
-  <?=mk_option($notify['date'], "Y-m-d", _("YYYY-MM-DD"))?>
-  </select>
+:notifications_duration_help:
 
-:notifications_date_format_help:
+_(Max notifications)_:
+: <input type="number" name="max" value="<?=$notify['max'] ?? 3?>" min="1" max="10">
 
-_(Time format)_:
-: <select name="time" class="a">
-  <?=mk_option($notify['time'], "h:i A", _("12 hours"))?>
-  <?=mk_option($notify['time'], "H:i", _("24 hours"))?>
-  </select>
+:notifications_max_help:
 
-:notifications_time_format_help:
 
+_(Auto-close)_ (_(seconds)_):
+: <input type="number" name="life" class="a" min="0" max="60" value="<?=$notify['life']?>"> _(a value of zero means no automatic closure)_
+
+:notifications_auto_close_help:
+
+
 _(Store notifications to flash)_:
 : <select name="path" class="a">
   <?=mk_option($notify['path'], "/tmp/notifications", _("No"))?>
   <?=mk_option($notify['path'], "/boot/config/plugins/dynamix/notifications", _("Yes"))?>
   </select>
