Index: /usr/local/emhttp/plugins/dynamix/Notifications.page
===================================================================
--- /usr/local/emhttp/plugins/dynamix/Notifications.page	original
+++ /usr/local/emhttp/plugins/dynamix/Notifications.page	modified
@@ -133,27 +133,11 @@
 _(Auto-close)_ (_(seconds)_):
 : <input type="number" name="life" class="a" min="0" max="60" value="<?=$notify['life']?>"> _(a value of zero means no automatic closure)_
 
 :notifications_auto_close_help:
 
-_(Date format)_:
-: <select name="date" class="a">
-  <?=mk_option($notify['date'], "d-m-Y", _("DD-MM-YYYY"))?>
-  <?=mk_option($notify['date'], "m-d-Y", _("MM-DD-YYYY"))?>
-  <?=mk_option($notify['date'], "Y-m-d", _("YYYY-MM-DD"))?>
-  </select>
 
-:notifications_date_format_help:
-
-_(Time format)_:
-: <select name="time" class="a">
-  <?=mk_option($notify['time'], "h:i A", _("12 hours"))?>
-  <?=mk_option($notify['time'], "H:i", _("24 hours"))?>
-  </select>
-
-:notifications_time_format_help:
-
 _(Store notifications to flash)_:
 : <select name="path" class="a">
   <?=mk_option($notify['path'], "/tmp/notifications", _("No"))?>
   <?=mk_option($notify['path'], "/boot/config/plugins/dynamix/notifications", _("Yes"))?>
   </select>
