Index: /usr/local/emhttp/plugins/dynamix/include/.set-password.php
===================================================================
--- /usr/local/emhttp/plugins/dynamix/include/.set-password.php	original
+++ /usr/local/emhttp/plugins/dynamix/include/.set-password.php	modified
@@ -410,7 +410,8 @@
             $el.addEventListener('keyup', () => {
                 if (displayValidation) debounce(validate()); // Wait until displayValidation is swapped in a change event
             });
         });
     </script>
+<?include "$docroot/plugins/dynamix.my.servers/include/welcome-modal.php"?>
 </body>
 </html>
