Index: /usr/local/emhttp/plugins/dynamix/include/Translations.php
===================================================================
--- /usr/local/emhttp/plugins/dynamix/include/Translations.php	original
+++ /usr/local/emhttp/plugins/dynamix/include/Translations.php	modified
@@ -73,11 +73,11 @@
   global $docroot,$language,$locale;
   $plugin = strtolower($plugin);
   $text = "$docroot/languages/$locale/$plugin.txt";
   if (file_exists($text)) {
     $store = "$docroot/languages/$locale/$plugin.dot";
-    if (!file_exists($store)) file_put_contents($store,serialize(parse_lang_file($text)));
+    clearstatcache(); if (!file_exists($store) || filemtime($text) > filemtime($store)) file_put_contents($store,serialize(parse_lang_file($text)));
     $language = array_merge($language,unserialize(file_get_contents($store)));
   }
 }
 
 // internal helper functions
@@ -131,11 +131,11 @@
 if ($locale) {
   $text = "$docroot/languages/$locale/translations.txt";
   if (file_exists($text)) {
     $store = "$docroot/languages/$locale/translations.dot";
     // global translations
-    if (!file_exists($store)) file_put_contents($store,serialize(parse_lang_file($text)));
+    clearstatcache(); if (!file_exists($store) || filemtime($text) > filemtime($store)) file_put_contents($store,serialize(parse_lang_file($text)));
     $language = unserialize(file_get_contents($store));
   }
   if (file_exists("$docroot/languages/$locale/helptext.txt")) {
     $root = "$docroot/languages/$locale/helptext.txt";
     $help = "$docroot/languages/$locale/helptext.dot";
@@ -161,17 +161,17 @@
 foreach($uri as $more) {
   $text = "$docroot/languages/$locale/$more.txt";
   if (file_exists($text)) {
     // additional translations
     $store = "$docroot/languages/$locale/$more.dot";
-    if (!file_exists($store)) file_put_contents($store,serialize(parse_lang_file($text)));
+    clearstatcache(); if (!file_exists($store) || filemtime($text) > filemtime($store)) file_put_contents($store,serialize(parse_lang_file($text)));
     $language = array_merge($language,unserialize(file_get_contents($store)));
   }
 }
 // help text
 if (($_SERVER['REQUEST_URI'][0]??'')=='/') {
-  if (!file_exists($help)) file_put_contents($help,serialize(parse_help_file($root)));
+  clearstatcache(); if (!file_exists($help) || filemtime($root) > filemtime($help)) file_put_contents($help,serialize(parse_help_file($root)));
   $language = array_merge($language,unserialize(file_get_contents($help)));
 }
 // remove unused variables
 unset($return,$jscript,$root,$help,$store,$uri,$more,$text);
 ?>
