diff --git a/node_modules/log4js/lib/layouts.js b/node_modules/log4js/lib/layouts.js
index 52aef56..5146063 100644
--- a/node_modules/log4js/lib/layouts.js
+++ b/node_modules/log4js/lib/layouts.js
@@ -137,7 +137,7 @@ function patternLayout(pattern, tokens) {
   const regex =
     /%(-?[0-9]+)?(\.?-?[0-9]+)?([[\]cdhmnprzxXyflosCMAF%])(\{([^}]+)\})?|([^%]+)/;
 
-  pattern = pattern || TTCC_CONVERSION_PATTERN;
+  pattern = typeof pattern === 'function' ? pattern() : (pattern || TTCC_CONVERSION_PATTERN);
 
   function categoryName(loggingEvent, specifier) {
     let loggerName = loggingEvent.categoryName;
